tests/integ/test_tuner.py

```diff
@@ -903,6 +903,7 @@ def test_tuning_byo_estimator(sagemaker_session, cpu_instance_type):
         predictor = tuner.deploy(1, cpu_instance_type, endpoint_name=best_training_job)
         predictor.serializer = _FactorizationMachineSerializer()
         predictor.deserializer = JSONDeserializer()
+        predictor.content_type = "application/json"
```

Comment:
In my next PR, I'm going to add serializer and deserializer parameters to `deploy`.